sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -59,6 +65,9 @@ def apply_CombinePerKey(self, transform, pcoll):
   def run(self, pipeline):
     """Execute the entire pipeline and returns an DirectPipelineResult."""
 
+    # Performing configured PTransform overrides.
+    pipeline.replace_all(DirectRunner.PTRANSFORM_OVERRIDES)
```

Comment:
Nit: I think this'd be less error prone (especially as we develop out more complicated inputs/outputs) if this was functional (i.e. returned a new pipeline object rather than mutating the existing one.